history.ts

```diff
@@ -0,0 +1,48 @@
+import { WeeklyArticleLog, WeeklyArticleHistory } from './types';
+import { get as getItem, set as setItem } from './lib/localStorage';
+import { getMondayFromDate } from './lib/dates';
+
+const historyWeeklyKey = 'gu.history.weeklyArticleCount';
+const mondayThisWeek = getMondayFromDate(new Date());
```

Comment:
I don't _think_ this will be an issue, but since this will only be happening once for a given pageview, is there a danger that it could become stale for example if the reader loaded the page right before midnight on a Sunday (also qualify that with - would it even matter if it did?)

Reply:
I can't see an issue here - no time for anything to go stale as we'd call this function and immediately update the local history on the user's browser. Unless I missed your point 🤔

Reply:
Yeah I think it's almost certainly not an issue. I meant that `mondayThisWeek` will get set when this module is loaded, and then it'll be used sometime later when we actually record an article view (which like you said we expect to happen very soon after).